cli/test-utils/src/util.rs

```diff
@@ -587,3 +594,126 @@ impl Retry {
         }
     }
 }
+
+pub struct ScriptTester<'a> {
+    pub accounts_pub: Vec<String>,
+    pub accounts_priv: Vec<String>,
+    pub provider: Provider<Http>,
+    pub nonces: BTreeMap<u32, U256>,
+    pub cmd: &'a mut TestCommand,
```

Comment:
given that this type works like a more sophisticated TestCommand, should we move ownership into this type?